polaris-migrator/README.md

```diff
@@ -119,6 +171,33 @@ In your PR, you can add a comment with the text `/snapit` to create a new [snaps
 npx @shopify/polaris-migrator@0.0.0-snapshot-release-20220919213536 replace-sass-function "./app/**/*.scss"
 ```
 
+### Checking migrations
+
+Running a migration can potentially modify thousands of files. For more complex migrations, a comment may be added suggesting the change is manually checked. You can quickly perform a manual search for this comment in your text editor:
+
+```
+polaris-migrator: This is a complex expression that we can't automatically convert. Please check this manually.
+```
+
+After applying a migration, it might be helpful to commit the changes that do not need a manual check from those that do. You can do this a few different ways, but we suggest staging all your changes, then unstaging those that include the manual check comment:
+
+```sh
+# Stage all modified files
+git add .
+
+# Unstage those that contain the manual check comment prefixed with "polaris-migrator:"
+git reset --mixed $(grep -r -l "polaris-migrator:")
```

Comment:
```suggestion
git reset $(grep -r -l "polaris-migrator:")
```

Do we need the `--mixed` option? The [git reset](https://git-scm.com/docs/git-reset#:~:text=If%20%3Cmode%3E%20is%20omitted%2C%20defaults%20to%20%2D%2Dmixed) docs suggest mixed mode is the default.